kml/serdes_gpx.cpp

```diff
@@ -30,6 +30,13 @@ std::string_view constexpr kDesc = "desc";
 std::string_view constexpr kMetadata = "metadata";
 std::string_view constexpr kEle = "ele";
 std::string_view constexpr kCmt = "cmt";
+
+std::string_view constexpr kGpxHeader =
```

Comment:
All these internal cpp file constants should be in the anonymous namespace to slightly reduce the binary size.

Reply:
Fixed